src/main/kotlin/org/jetbrains/research/testspark/tools/TestProcessor.kt

```diff
@@ -140,7 +140,7 @@ class TestProcessor(
 
         CommandLineRunner.run(command as ArrayList<String>)
 
-        return testExecutionError
+        return if (testExecutionResult.isSuccessful()) "" else testExecutionResult.executionMessage
```

Comment:
It is advisable to change the API of this `TestProcessor`'s method to return `testExecutionResult`. I guess it requires a lot of refactoring; therefore, it is not for this PR.

Leave a TODO to change the method's return contract.